geopy/util.py

```diff
@@ -80,3 +80,10 @@ def decode_page(page):
         else: # requests?
             encoding = page.headers.get("charset") or "utf-8"
             return str(page.content, encoding=encoding)
+
+
+def get_version():
```

Comment:
Just that it can be `"geopy/%s" % geopy.version.GEOPY_VERSION` or the like, determined at interpreter start. No need for a function call.